src/compile_torture_tests.py

```diff
@@ -82,7 +89,7 @@ def run(c, cxx, testsuite, sysroot_dir, fails, out, config='wasm'):
   return result
 
 
-def getargs():
```

Comment:
I think if `main` were doing more things I'd feel more strongly about it. As it is `main` gets the arguments, delegates to the runner, and returns the exit code.

And +1 to consistency everywhere in https://github.com/WebAssembly/waterfall/pull/188